README.md

```diff
@@ -5,8 +5,10 @@ This repository stores all the source files of TiDB in Kubernetes documentation.
 - `en`: [documentation in English](/en/TOC.md)
 - `zh`: [documentation in Chinese](/zh/TOC.md)
 
-> **Note:**
->
-> - Currently, this repository stores the TiDB Operator v1.1 documentation directly on the `master` branch, which is scheduled to be moved to the `release-1.1` branch by the end of March, 2020.
->
-> - v1.0 documentation will also be moved here soon. Before that, you can see [v1.0 documentation](https://pingcap.com/docs/v3.0/tidb-operator-overview/) at the PingCAP website.
+## Documentation versions
+
+Currently, TiDB in Kubernetes maintains the following three versions of documentation, each in a separate branch:
+
+    - `master`: the latest version in development
```

Comment:
The indentation here makes it display like a code block. You can refer to: https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#learn-about-docs-versions

Reply:
Updated.